webapp/src/main/webapp/news-list-view/components/views/NewsLatestView.vue

```diff
@@ -196,6 +205,20 @@ export default {
         members: this.$t('space.members'),
       };
     },
+    spaceUrl() {
+      let groupId;
+      if (this.space && this.space.groupId) {
+        groupId = this.space.groupId.replace(/\//g, ':');
+        return `${eXo.env.portal.context}/g/${groupId}/`;
+      }
+      return `${eXo.env.portal.context}/dw/home/`;
+    },
```

Comment:
we should get the space url directly from this.space.url, then the site url should not be explicit with the site name